composeApp/src/commonMain/kotlin/designsystem/login/LoginFooter.kt

```diff
@@ -0,0 +1,36 @@
+package designsystem.login
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import designsystem.colors.TextColor
+import dev.icerock.moko.resources.compose.stringResource
+import instaclone.resources.MR
+
+@Composable
+fun LoginFooter() {
```

Comment:
I think we also should add the modifier here

Reply:
yep!